packages/user/src/supertokens/recipes/config/third-party-email-password/emailPasswordSignUp.ts

```diff
@@ -10,6 +10,14 @@ const emailPasswordSignUp = (
   const { config, log } = fastify;
 
   return async (input) => {
+    if (config.user.supertokens.features?.signUp) {
+      throw {
```

Comment:
this condition will only satisfy if only the singUp is set to true. do we want to disable signup if `feature.signUp: true`?